.eslintrc

```diff
@@ -0,0 +1,35 @@
+env:
+  node: true
+  browser: true
+
+rules:
+  brace-style: 2
+  comma-spacing: 1
+  eol-last: 0
+  indent: [2, 2]
```

Comment:
The reason I prefer spaces is that tabs have no globally defined width.
Consider this:

```js
function() {
  var one = 1,
      two = 2;
  // ...
}
```

If you do this with tabs, it looks like this:
```js
function() {
<tab>var one = 1,
<tab>    two = 2;
  // ...
}
```

Using tabs *and* spaces in the same project is absolutely ridiculous.  
Any decent editor will allow you to insert *n* spaces when you press tab.  
2 spaces is also what's used pretty much everywhere in the node community.

----

What do the others think about this?